spec/features/index_feature_spec.rb

```diff
@@ -0,0 +1,46 @@
+feature 'Webpages loading correctly' do
+  scenario 'Webpage returns status code 200' do
+    visit '/'
+    expect(page.status_code).to eq(200)
+  end
+
+  scenario 'Displays all text as expected' do
+    visit '/'
+    expect(page).to have_content('Rock Paper Scissors', 'Enter name below to start:')
+  end
+
+  scenario 'Displays name text field' do
+    visit '/'
+    expect(page).to have_field('name')
```

Comment:
Very clear and efficient code, tests aren't cluttered and scenarios clearly describe what is expected